### DISCUSSION TITLE
CSSBattle Learn - Lvl 61

### DISCUSSION THREAD

**Discussion by Microbz** [cssbattle/forum#309] [Q&A] [answered]
Hello, 

I'm trying to complete [Level 61 of CSSBattle Learn](https://cssbattle.dev/learn/61).

![image](https://user-images.githubusercontent.com/7910178/192113078-dba6bf2f-4345-4785-845b-af3098f69e50.png)

![image](https://user-images.githubusercontent.com/7910178/192113089-d8f4569b-cdba-45dd-a3aa-b71dc106fb39.png)

I don't understand what's wrong ... 
I tried with or without spread value.

Can you help me please ?

**Comment by aarthvaen** [comment, marked as answer]
No need of `filter` when applying the opacity property. `opacity: <number/percentage>` is a more established and used syntax. Therefore, only writing `opacity: 50%;` should do the job.

**Reply by Microbz** [reply]
Thanks a lot !
